Debug/Scripts/x86_64_target_definition.py

```diff
@@ -196,11 +198,6 @@ def get_reg_num(reg_num_dict, reg_name):
     return LLDB_INVALID_REGNUM
 
 
-def get_reg_num(reg_num_dict, reg_name):
```

Comment:
Why was that removed, sorry?

Reply:
Weird! I think I missed it but in my master branch, `get_reg_num` is there: [source](https://github.com/dreamwhite/OpenCorePkg/blob/master/Debug/Scripts/x86_64_target_definition.py)

Reply:
Well, it is clearly deleted in the PR though.

Reply:
Ouch, I'm kinda confused at this point. PR says the function is not there, but the master branch has it 😅